src/formatDate.js

```diff
@@ -50,7 +50,38 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArray = date.split(fromFormat[fromFormat.length - 1]);
+
+  const result = [];
+
+  for (let i = 0; i < toFormat.slice(0, 3).length; i++) {
+    const value = toFormat[i];
+    let currentValue = value;
+
+    if (value === 'YYYY') {
+      currentValue = 'YY';
+    }
+
+    const fromIndex = fromFormat
+      .findIndex((fromValue) => fromValue.includes(currentValue));
+    let newValue = dateArray[fromIndex];
+
+    if (value === 'YYYY' && newValue.length === 2) {
+      if (newValue < 30) {
+        newValue = '20' + newValue;
```

Comment:
You can use interpolation here
```suggestion
        newValue = `20${newValue}`;
```